### ISSUE TITLE
Search screen accessibility bugs

### ISSUE THREAD

**Issue by Rajat-Sen** [StatCan/StatcanPublicationUAT#430]
Related to #239 

Search screen (First time)
1. On the search screen, when swiping from left to right, the focus should not go to the image inside the search field. This applies to Recent searches, when the user is searching and when there are no results to be displayed.
2. There should be no Focus on the decorative image.
3. Cross icon should be labelled as "clear text icon" button as it is saying button two times.
4. When searching, clicked on filter by subjects, the Focus should go to "Filter by subject(s)" heading > Close button
5. When the Focus goes to a particular subject, it should be say the subject name and the state of the checkbox(Checked/unchecked).

**Comment by Rajat-Sen** [comment]
Closing this ticket for now as we are going to change it.
